### PR TITLE
Update play.js

### PR DESCRIPTION
**Pull request by Bacon-Fixation**
Addressing #373
Jriskamm found that mobile links would not work

Much Love
-Bacon

### CODE REVIEW

**Comment by galnir** [comment]
Web youtube links don't pass the if statement with your solution
https://www.youtube.com/watch?v=dQw4w9WgXcQ for example

**Comment by Bacon-Fixation** [comment]
image

i don't get the same result

**Comment by Bacon-Fixation** [comment]
Great Song Btw

**Comment by Jriskamm** [comment]
> Web youtube links don't pass the if statement with your solution
> https://www.youtube.com/watch?v=dQw4w9WgXcQ for example

it works for me too

**Comment by Bacon-Fixation** [comment]
Jriskamm is also using this Copy of Play.js a swell He is the one that ran a crossed it

Much Love
-Bacon

File: commands/music/play.js [hunk 1]
```diff
@@ -202,11 +202,10 @@ module.exports = class PlayCommand extends Command {
...
     }
-    if (query.match(/^(http(s)?:\/\/)?(m.)?youtu(be|.be)?(\.com)?\/.+/))
-      query = query
-        .replace(/https:\/\/m./g, `https://www.`)
-        .replace(/http:\/\/m./g, `http://www.`);
-    if (query.match(/^(http(s)?:\/\/)?((w){3}.)?youtu(be|.be)?(\.com)?\/.+/)) {
+
+    if (
+      query.match(/^(http(s)?:\/\/)?(m.)?((w){3}.)?youtu(be|.be)?(\.com)?\/.+/)
+    ) {
```
Comment (Bacon-Fixation) [inline commands/music/play.js]: i think this is more what you were thinking right ? both work but this is better

Comment (Bacon-Fixation) [inline commands/music/play.js]: i think this is more what you were thinking right ?
Both work but this is better

File: commands/music/play.js [hunk 2]
```diff
@@ -51,82 +51,86 @@ module.exports = class PlayCommand extends Command {
...
-    if (db.get(message.member.id) !== null) {
-      const userPlaylists = db.get(message.member.id).savedPlaylists;
-      let found = false;
-      let location;
-      for (let i = 0; i < userPlaylists.length; i++) {
-        if (userPlaylists[i].name == query) {
-          found = true;
-          location = i;
-          break;
+    if (
```
Comment (galnir) [inline commands/music/play.js:54]: Why did you add this 'if' statement here?

Comment (Bacon-Fixation) [reply commands/music/play.js:54]: only lines 205 - 208 were Changed

Rest of play.js was formatted Via the settings that were set inside the `.prettierrc` file

**Review by galnir — APPROVED** [review]
Great, thanks for addressing #373 👍
